src/frontend/vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+})
```

Comment:
TODO we'll need to configure this setting for sure,

https://vitejs.dev/config/server-options.html#server-proxy
https://vitejs.dev/config/server-options.html#server-origin

Reply:
Yep for sure